File: mcp/scripts/delete_user.php
<?php
require_once __DIR__ . '/../config/firebase/config.php';

// 삭제할 회원의 전화번호
$phoneNumber = '+821012341234'; // 우리집탄이의 전화번호 (국제 형식)

try {
    // 전화번호로 사용자 검색
    $userRecord = $auth->getUserByPhoneNumber($phoneNumber);
    
    if ($userRecord) {
        // 사용자 삭제
        $auth->deleteUser($userRecord->uid);
        echo "회원이 성공적으로 삭제되었습니다.\n";
        
        // 추가 데이터가 있다면 여기서 삭제
        $database->getReference('users/' . $userRecord->uid)->remove();
        echo "회원 데이터가 성공적으로 삭제되었습니다.\n";
    } else {
        echo "해당 전화번호로 등록된 회원이 없습니다.\n";
    }
} catch (Exception $e) {
    echo "오류 발생: " . $e->getMessage() . "\n";
    
    // 오류 상세 정보 출력
    if ($e instanceof \Kreait\Firebase\Exception\AuthException) {
        echo "Firebase Auth 오류: " . $e->getMessage() . "\n";
    } elseif ($e instanceof \Kreait\Firebase\Exception\DatabaseException) {
        echo "Firebase Database 오류: " . $e->getMessage() . "\n";
    }
} 

File: mcp/scripts/delete_user.py
import firebase_admin
from firebase_admin import credentials, auth, db
import json
import os

# 서비스 계정 키 파일 경로
cred = credentials.Certificate('config/google/service-account.json')

# Firebase 초기화
firebase_admin.initialize_app(cred, {
    'databaseURL': 'https://topmkt-832f2-default-rtdb.firebaseio.com'
})

# 삭제할 사용자의 전화번호
phone_number = '+821012341234'  # 우리집탄이의 전화번호

try:
    # 전화번호로 사용자 검색
    user = auth.get_user_by_phone_number(phone_number)
    
    if user:
        # 사용자 삭제
        auth.delete_user(user.uid)
        print(f"사용자 {user.uid}가 성공적으로 삭제되었습니다.")
        
        # 추가 데이터가 있다면 삭제
        ref = db.reference(f'users/{user.uid}')
        ref.delete()
        print("사용자 데이터가 성공적으로 삭제되었습니다.")
    else:
        print("해당 전화번호로 등록된 사용자가 없습니다.")

except Exception as e:
    print(f"오류 발생: {str(e)}") 